### PR TITLE
generate missing rust code

### PR DESCRIPTION
**Pull request by BusyJay**
@coocood @shenli PTAL

### CODE REVIEW

File: go-tipb/expression.pb.go [hunk 1]
```diff
@@ -33,6 +33,10 @@ var _ = proto.Marshal
 var _ = fmt.Errorf
 var _ = math.Inf
 
+// This is a compile-time assertion to ensure that this generated file
+// is compatible with the proto package it is being compiled against.
+const _ = proto.ProtoPackageIsVersion1
```
Comment (BusyJay) [inline go-tipb/expression.pb.go]: I'm not sure if this line is appropriate.

**Comment by coocood** [comment]
LGTM

File: go-tipb/expression.pb.go (hunk 1 above)
Comment (shenli) [inline go-tipb/expression.pb.go]: Seems this line is deleted in this PR: https://github.com/pingcap/tipb/pull/6/files
Maybe we use different protoc.

**Comment by shenli** [comment]
LGTM

**Pull request merged**